src/test/java/com/artipie/management/api/artifactory/AddUpdatePermissionSliceTest.java

```diff
@@ -78,12 +80,13 @@ void returnsBadRequestOnInvalidRequest() {
     }
 
     @Test
+    @Disabled
```

Comment:
@olenagerasimova could you please explain why this test is disabled? If it is because `FakeRepoPerms` does not support some methods yet, then I'd suggest to use `RepoPermissions.FromSettings` class in `AddUpdatePermissionSlice` for now and delete this class later. It is too easy to forget to enable the test later when it starts working